packages/payment-detection/test/erc20/escrow-info-retriever.test.ts

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable no-invalid-this */
+/* eslint-disable no-magic-numbers */
+import { PaymentTypes } from '@requestnetwork/types';
+import EscrowERC20InfoRetriever from '../../src/erc20/escrow-info-retriever';
+import ProxyERC20InfoRetriever from '../../src/erc20/proxy-info-retriever';
+import { ethers } from 'ethers';
+
+const erc20LocalhostContractAddress = '0x9FBDa871d559710256a2502A2517b794B482Db40';
+const escrowContractAddress = '0xF08dF3eFDD854FEDE77Ed3b2E515090EEe765154';
+const proxyContractAddress = '0x2C2B9C9a4a25e24B174f26114e8926a9f2128FE4';
+const feeProxyContractAddress = '0x75c35C980C0d37ef46DF04d31A140b65503c0eEd';
+const paymentReferenceMock = 'aaaa';
+const feeAddress = 'c5fdf4076b8f3a5357c5e395ab970b5b54098fef';
+const paymentAddress = '0xf17f52151EbEF6C7334FAD080c5704D77216b732';
+
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+describe('api/erc20/escrow-info-retriever', () => {
```

Comment:
Missing tests: the 3 events that are specific to escrow

Reply:
done